temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
```

Comment:
Avoid using `pdb` in committed code, as it is meant for debugging purposes and should not be present in production code.
```
Remove the import statement for `pdb`.
```